content/drone-plugins/drone-ansible/index.md

```diff
@@ -0,0 +1,175 @@
+---
+version: '1.0'
+date: 2019-02-15T00:00:00+00:00
+title: Ansible
+author: tboerger
```

Comment:
drone-plugins, it's a community managed plugin :)

Reply:
...and fixed.